helm/prometheus-rules/templates/alerting-rules/areas/kaas/bigmac/deployment.management-cluster.rules.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+{{- if not .Values.mimir.enabled }}
+    cluster_type: "management_cluster"
+{{- end }}
+  name: deployment.management-cluster.rules
```

Comment:
I Guess you Split one file hère. You probably need to prefix thé rule name with the team name ?